home/config/niri.nix

```diff
@@ -0,0 +1,136 @@
+{ pkgs, lib, ... }:
+let
+  mod = "Mod4";
+  term = "wezterm";
+  lockcmd = "${pkgs.swaylock}/bin/swaylock -fF";
+
+  sounds = pkgs.stdenv.mkDerivation {
+    name = "nixfiles-sounds";
+    src = ../../assets/sounds;
+    installPhase = ''
+      mkdir -p $out
+      cp -r ./* $out/
+    '';
+  };
+
+  push-to-talk = pkgs.writeScript "push-to-talk" ''
+    case $1 in
+        on)
+            pamixer --default-source -u
+            pw-cat -p "${sounds}/ptt-activate.mp3"
+        ;;
+        off)
+            pamixer --default-source -m
+            pw-cat -p "${sounds}/ptt-deactivate.mp3"
+        ;;
+    esac
+  '';
```

Comment:
Can you pull these out into a separate nix file so they can be reused between the sway and niri configs?